daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/BitFunctions.tdml

```diff
@@ -0,0 +1,68 @@
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<tdml:testSuite suiteName="SimpleTypes" description="Section 5 - simple types"
+  xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions"
+  xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+  xmlns:ex="http://example.com"
+  xmlns:home="http://home.com"
+  xmlns:first="http://first.com"
+  defaultRoundTrip="onePass">
+  <tdml:defineSchema elementFormDefault="unqualified" name="BitFunctions">
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+    <dfdl:format ref="ex:GeneralFormat"/>
+
+       <xs:element name="round08" type="xs:int" dfdl:inputValueCalc="{ dfdlx:leftShift( 5,2)}"/>
+        <xs:element name="round09" type="xs:int" dfdl:inputValueCalc="{ fdfdlx:leftShift( 9,3) }"/>
+        <xs:element name="round10">
+          <xs:complexType>
+            <xs:sequence>
+              <xs:element name="val" type="xs:int" dfdl:inputValueCalc="{ dfdlx:leftShift( 9,3)}"/>
+              <xs:element name="value" type="xs:int" dfdl:inputValueCalc="{dfdlx:leftShift( 7,4)}"/>
+            </xs:sequence>
+          </xs:complexType>
+        </xs:element>
+
+        <xs:element name="leftInt">
+          <xs:complexType>
+            <xs:sequence dfdl:separator=",">
+              <xs:element name="num" type="xs:int" dfdl:lengthKind="delimited"/>
+              <xs:element name="count" type="xs:int" dfdl:lengthKind="delimited"/>
+              <xs:element name="value" type="xs:int" dfdl:inputValueCalc="{ dfdlx:leftShift( num,count) }"/>
+            </xs:sequence>
+          </xs:complexType>
+        </xs:element>
+          <tdml:parserTestCase name="BitFunctions_leftShift_int_01" root="leftInt"
+              model="BitFunctions" >
+              <tdml:document>
+                1,2
+              </tdml:document>
+              <tdml:infoset>
+                <tdml:dfdlInfoset>
+                 <ex:leftShift>
+                 <num>7</num>
+                 <count>2</count>
+                  <value>4</value>
+                    </ex:leftShift>
+                </tdml:dfdlInfoset>
+              </tdml:infoset>
+            </tdml:parserTestCase>
+            </tdml:testSuite>
```

Comment:
Pleaes double check the indentation in this file. The parserTestCase and closing testSuite lines aren't indented correctly.